streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/KStreamTest.scala

```diff
@@ -85,12 +85,34 @@ class KStreamTest extends FlatSpec with Matchers with TestDriver {
     val testDriver = createTestDriver(builder)
 
     testDriver.pipeRecord(sourceTopic, ("1", "value1"))
+    acc shouldBe "value1"
```

Comment:
Sorry for my OCD but could you add a new line after this so that we see it's different steps of the test?